fix whitelist baserow and add comment for URL (#8)
kendraio/kendraio-proxy: conf/destinationAllowlist.txt

>   # One url per line 

Reviewer: ```suggestion
  # One hostname per line, NOT a URL path 
```

  # Any content after the first space is removed (So make sure there is no space in front of your url)

>   # URL must be without http or https as it will be added automatically

Reviewer: ```suggestion
  # The rule checker expects a valid hostname only, not a full path - it must not specify http or https protocol.
```

disease.sh # New Covid-19 data source
www.googleapis.com # For YouTube and Google Drive etc
kendraio-dev-shop.myshopify.com # Shopify Store for testing purpose

> api.baserow.io/api # baserow db API

Reviewer: ```suggestion
api.baserow.io # Baserow database API
```
